Source/SuperMario3DWorld/rules.txt

```diff
@@ -176,19 +343,16 @@ formats = 0x816
 overwriteWidth = ($width/$gameWidth) * 64
 overwriteHeight = ($height/$gameHeight) * 32
 
+[TextureRedefine]
+width = 60
+height = 33
+formats = 0x816
+overwriteWidth = ($width/$gameWidth) * 60
+overwriteHeight = ($height/$gameHeight) * 33
 
-# Depth of field, enabling breaks game
-#[TextureRedefine]
-#width = 640
-#height = 368
-#formats = 0x816
-#overwriteWidth = ($width/$gameWidth) * 640
-#overwriteHeight = ($height/$gameHeight) * 368
-
-#[TextureRedefine]
-#width = 512
-#height = 256
-#formats = 0x816
-#overwriteWidth = ($width/$gameWidth) * 512
-#overwriteHeight = ($height/$gameHeight) * 256
-
+[TextureRedefine]
+width = 40
+height = 22
+formats = 0x816
+overwriteWidth = ($width/$gameWidth) * 60
+overwriteHeight = ($height/$gameHeight) * 33
```

Comment:
```
overwriteWidth = ($width/$gameWidth) * 40
overwriteHeight = ($height/$gameHeight) * 22

Reply:
Fixed all copy paste ***** and white space